bigbluebutton-html5/imports/ui/components/media/container.jsx

```diff
@@ -127,6 +131,26 @@ export default withLayoutConsumer(withModalMounter(withTracker(() => {
     data.children = <ScreenshareContainer />;
   }
 
+  const userIsInBreakout = breakoutService.getBreakoutUserIsIn(Auth.userID);
+  let deviceIds = Session.get('deviceIds');
+
+  if (!userIsInBreakout && userWasInBreakout && deviceIds && deviceIds !== '') {
+    const canConnect = Session.get('canConnect');
+    deviceIds = deviceIds.split(',');
+
+    if (canConnect) {
```

Comment:
Hey. What's the rationale behind tethering joinVideo to this `canConnect` flag (here and in video-list-item's onloadeddata callback)?

Reply:
It is needed in case the user has more than one active camera (can't share more than one at the same time), so we only share the second camera after the first has finished loading

Reply:
Ok. Could you please add a comment there explaining this? For the sake of posterity.

Reply:
Sure! Just added the comments (commit 4a761f9)